### ISSUE TITLE
CRAN doc fixes

### ISSUE THREAD

**Issue by tdhock** [animint/animint2#162]
from https://cran.r-project.org/web/checks/check_results_animint2.html
```
Version: 2024.1.24
Check: Rd cross-references
Result: NOTE
  Found the following Rd file(s) with Rd \link{} targets missing package
  anchors:
    ggplot_gtable.Rd: gtable
  Please provide package anchors for all Rd \link{} targets not in the
  package itself and the base packages.
Flavors: [r-devel-linux-x86_64-debian-clang](https://www.r-project.org/nosvn/R.check/r-devel-linux-x86_64-debian-clang/animint2-00check.html), [r-devel-linux-x86_64-debian-gcc](https://www.r-project.org/nosvn/R.check/r-devel-linux-x86_64-debian-gcc/animint2-00check.html), [r-devel-windows-x86_64](https://www.r-project.org/nosvn/R.check/r-devel-windows-x86_64/animint2-00check.html)

Version: 2024.1.24
Check: HTML version of manual
Result: NOTE
  Found the following HTML validation problems:
  stat_unique.html:110:1 (stat_unique.Rd:65): Warning: trimming empty <li>
  stat_unique.html:109:1 (stat_unique.Rd:65): Warning: trimming empty <ul>
```
need to fix before Nov 12

**Comment by tdhock** [comment]
build ok on windows R-release
```
$ bash build.sh 

R version 4.4.1 (2024-06-14 ucrt) -- "Race for Your Life"
Copyright (C) 2024 The R Foundation for Statistical Computing
Platform: x86_64-w64-mingw32/x64

R is free software and comes with ABSOLUTELY NO WARRANTY.
You are welcome to redistribute it under certain conditions.
Type 'license()' or 'licence()' for distribution details.

  Natural language support but running in an English locale

R is a collaborative project with many contributors.
Type 'contributors()' for more information and
'citation()' on how to cite R or R packages in publications.

Type 'demo()' for some demos, 'help()' for on-line help, or
'help.start()' for an HTML browser interface to help.
Type 'q()' to quit R.

> devtools::document()
ℹ Updating animint2 documentation
Writing 'NAMESPACE'
ℹ Loading animint2
Writing 'NAMESPACE'
Writing 'animint2dir.Rd'
Writing 'animint2pages.Rd'
> 
> 

built animint2_2024.10.11.tar.gz so now we INSTALL
* installing to library 'C:/Users/hoct2726/AppData/Local/R/win-library/4.4'
* installing *source* package 'animint2' ...
** using staged installation
** R
** data
*** moving datasets to lazyload DB
** inst
** byte-compile and prepare package for lazy loading
** help
*** installing help indices
*** copying figures
** building package indices
** testing if installed package can be loaded from temporary location
** testing if installed package can be loaded from final location
** testing if installed package keeps a record of temporary installation path
* DONE (animint2)
* using log directory 'C:/Users/hoct2726/R/animint2.Rcheck'
* using R version 4.4.1 (2024-06-14 ucrt)
* using platform: x86_64-w64-mingw32
* R was compiled by
    gcc.exe (GCC) 13.2.0
    GNU Fortran (GCC) 13.2.0
* running under: Windows 11 x64 (build 22631)
* using session charset: UTF-8
* using option '--as-cran'
* checking for file 'animint2/DESCRIPTION' ... OK
* this is package 'animint2' version '2024.10.11'
* package encoding: UTF-8
* checking CRAN incoming feasibility ... [80s] Note_to_CRAN_maintainers
Maintainer: 'Toby Hocking <toby.hocking@r-project.org>'
* checking package namespace information ... OK
* checking package dependencies ... OK
* checking if this is a source package ... OK
* checking if there is a namespace ... OK
* checking for executable files ... OK
* checking for hidden files and directories ... OK
* checking for portable file names ... OK
* checking whether package 'animint2' can be installed ... [19s] OK
* checking installed package size ... OK
* checking package directory ... OK
* checking for future file timestamps ... OK
* checking DESCRIPTION meta-information ... OK
* checking top-level files ... OK
* checking for left-over files ... OK
* checking index information ... OK
* checking package subdirectories ... OK
* checking code files for non-ASCII characters ... OK
* checking R files for syntax errors ... OK
* checking whether the package can be loaded ... OK
* checking whether the package can be loaded with stated dependencies ... OK
* checking whether the package can be unloaded cleanly ... OK
* checking whether the namespace can be loaded with stated dependencies ... OK
* checking whether the namespace can be unloaded cleanly ... OK
* checking loading without being on the library search path ... OK
* checking whether startup messages can be suppressed ... OK
* checking use of S3 registration ... OK
* checking dependencies in R code ... OK
* checking S3 generic/method consistency ... OK
* checking replacement functions ... OK
* checking foreign function calls ... OK
* checking R code for possible problems ... [24s] OK
* checking Rd files ... OK
* checking Rd metadata ... OK
* checking Rd line widths ... OK
* checking Rd cross-references ... OK
* checking for missing documentation entries ... OK
* checking for code/documentation mismatches ... OK
* checking Rd \usage sections ... OK
* checking Rd contents ... OK
* checking for unstated dependencies in examples ... OK
* checking contents of 'data' directory ... OK
* checking data for non-ASCII characters ... OK
* checking LazyData ... OK
* checking data for ASCII and uncompressed saves ... OK
* checking examples ... [64s] OK
* checking examples with --run-donttest ... [127s] OK
* checking for unstated dependencies in 'tests' ... OK
* checking tests ...
  Running 'namespace.R'
  Running 'testthat.R' [104s]
 [107s] OK
* checking PDF version of manual ... [17s] OK
* checking HTML version of manual ... [39s] OK
* checking for non-standard things in the check directory ... OK
* checking for detritus in the temp directory ... OK
* DONE

Status: OK
```

**Comment by tdhock** [comment]
looks like this problem is fixed using the new version 2024.11.2 which I put on CRAN a few days ago.
